web/backend/src/controllers/users.ts

```diff
@@ -34,14 +34,28 @@ usersRouter.post('/add_role', (req, res, next) => {
     }
   }
 
-  addPolicy(req.body.userId, req.body.subject, req.body.permission)
-    .then(() => {
-      res.set(200).send();
-      next();
-    })
-    .catch((e) => {
-      res.status(400).send(`Error while adding to roles: ${e}`);
-    });
+  if (Object.hasOwn(req.body, 'userId')) {
```

Comment:
For next time: I think `'userId' in req.body` would be clearer to me. Same below.